atest/robot/standard_libraries/builtin/run_keyword_with_errors.robot

```diff
@@ -111,6 +126,21 @@ Expect Error When Syntax Error In Setting Variables
 Expect Error When Syntax Error In For Loop
     Check Test Case    ${TEST NAME}
 
+Expect Error When Access To Nonexisting Variable
+    Check Test Case    ${TEST NAME}
+
+Expect Error When Access To List Variable Nonexisting Index Syntax 1
+    Check Test Case    ${TEST NAME}
+
+Expect Error When Access To List Variable Nonexisting Index Syntax 2
```

Comment:
It's fine to have multiple tests on the test data side for one feature named like `Something 1`, `Something 2`, etc. They should, however, be combined on the execution side:

```robotframework
*** Test Cases ***
Something
    Check Test Case   ${TEST NAME} 1
    Check Test Case   ${TEST NAME} 2
```